hail/python/hailctl/dataproc/connect.py

```diff
@@ -54,12 +54,25 @@ def main(args, pass_through_args):
         stderr=sp.STDOUT
     )
 
-    for c in ['chromium', 'chromium-browser']:
-        chrome = which(c)
-        if chrome is not None:
-            break
-    if chrome is None:
+    import platform
+    system = platform.system()
+
+    if system == 'Darwin':
         chrome = r'/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+        user_data_dir = '/tmp/'
```

Comment:
Instead of these constants, we could maybe use [`tempfile.gettempdir`](https://docs.python.org/3/library/tempfile.html?highlight=tempdir#tempfile.gettempdir)?

Reply:
oh, good suggestion.